index.html

```diff
@@ -88,7 +90,9 @@ <h2 class="heading-mid-heavy">
                     <li class="program program-section">
                         <i class="bi bi-easel fa-2x"></i>
                         <span>Talk</span>
-                        <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Soluta, beatae.</p>
+                        <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Soluta, beatae
+                            consectetur adipisicing elit. Soluta, beatae.
+                        </p>
                     </li>
                     <li class="program program-section">
                         <i class="bi bi-shop-window fa-2x"></i>
```

Comment:
- [ ] Great job so far. I am unable to highlight the speaker part I wanted to leave comment on for some reason. I decided to highlight this part and leave the comment with screenshots that would carry my message to you.
- [ ] It is required to produce a close replica of original design with customized text and images. The number of items at a section should be maintained. There're six featured speakers on the desktop(home page) on the original design, but your design carries four featured speakers. Kindly add the remaining two.

#### Your Design
![your-speaker-desktop](https://github.com/Paulinelali/Microverse-capstone-one/assets/26597432/96e46ad4-3d23-4546-8d24-79e58ab6b862)

### Original Design
![temp-speaker-desktop](https://github.com/Paulinelali/Microverse-capstone-one/assets/26597432/92e5b027-f283-42ef-b75d-199047aca463)


- [ ] On the mobile, the title `Featured Speakers` in missing on your design. Kindly look into it and fix.

### Your Design
![your-speaker-display-mobile](https://github.com/Paulinelali/Microverse-capstone-one/assets/26597432/8e2f7af2-5d8a-4625-be9b-833e1b332896)

### Original Design

![temp-speaker-display-mobile](https://github.com/Paulinelali/Microverse-capstone-one/assets/26597432/7b6e4c64-fb4b-4621-a34d-87911c0cd0da)


- [ ] On the mobile too, `SEE THE WHOLE PROGRAM` text is not visible on the mobile on the original design, but kind of shows and not properly position on your design. Kindly remove it. It is not meant to be visible on the mobile(home page).

### Your Design
![your-extral-text-mobile](https://github.com/Paulinelali/Microverse-capstone-one/assets/26597432/353da120-882b-4a97-8cf8-40dc0943774e)

### Original Design

![temp-extral](https://github.com/Paulinelali/Microverse-capstone-one/assets/26597432/2494ae14-a37c-4c9e-b8e7-8f741dabd968)